ansible/roles/davros/files/authproxy/authproxy.py

```diff
@@ -67,6 +67,7 @@ def callback():
             'code': flask.request.args['code'],
         },
         auth=(config['LIQUID_CLIENT_ID'], config['LIQUID_CLIENT_SECRET']),
+        allow_redirects=False,
```

Comment:
Howcome? Should we add this to [hoovere oauth](https://github.com/hoover/search/blob/b0736b41/hoover/contrib/oauth2/views.py#L32) also?

Reply:
Don't know what it doesn, i just merged it from `demo`. What are the implications of this?

Reply:
Can't remember. I say, let's remove it, and if it was needed, we'll add it back.